### DISCUSSION TITLE
Some questions about the kernel_iph plugin

### DISCUSSION THREAD

**Discussion by wuliangxiaoseng** [strongswan/strongswan#1667] [Q&A]
The backend does not support installation of [virtual IP] addresses for IPsec clients. It says so in the documentation.  
I wanted to do this with reference to the Linux version, but I found that the data structure definitions were very different, so what was the reason for this design.

**Comment by tobiasbrunner** [comment]
Sorry, I don't fully understand what are asking. The _kernel-iph_ plugin is for Windows, not Linux. And yes, it does currently not support installing virtual IPs, because these don't work on Windows (see the last commit in the _win-vip_ branch for details). This might change in the future (with the help of Wintun, @strongX509  was working on that).
